### PR TITLE
Add FAQ "How do I disable a rule?"

### LINKED ISSUE #1922: Add "How do I disable a rule" to FAQ
This question comes up all the time (e.g. [just now](https://twitter.com/andreysitnik/status/778911556872667136)). We should maybe even put it at the top of FAQ.

### PR DESCRIPTION
**Pull request by m-allanson**
Closes #1922

### CODE REVIEW

File: docs/user-guide/faq.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # FAQ
 
+## How do I disable a rule?
+
+You can disable a rule by setting its config value to `null`
```
Comment (davidtheclark) [inline docs/user-guide/faq.md]: Missing period.

File: docs/user-guide/faq.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+For example, to use `stylelint-config-standard` without the `at-rule-empty-line-before` rule:
+```
```
Comment (davidtheclark) [inline docs/user-guide/faq.md]: Can we add `json` here for syntax highlighting?

File: docs/user-guide/faq.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "at-rule-empty-line-before": null
+  }
+}
+```
+
+You can also disable a rule from within your CSS. Refer to the rules section of the [configuration guide](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#rules) for more information.
```
Comment (davidtheclark) [inline docs/user-guide/faq.md]: How about changing "from within your CSS" to "for specific sections of your CSS"?

**Comment by m-allanson** [comment]
PR updated to cover those three requests.

**Comment by jeddy3** [comment]
LGTM. Thanks @m-allanson !

**Pull request merged**